tests/test_document.py

```diff
@@ -526,9 +535,6 @@ def _get_mini_doc(self,):
         doc.creation_info.add_creator(Tool('ScanCode'))
```

Comment:
hmm, in `tv-mini.tv` no creator is set... Is this not part of the test?

Reply:
Good catch. This is actually not part of the test at the moment. I will delete the creator as it is not needed for writing (better: does not lead to a failure of the test). The creation date must be set because the tv and rdf-writer would fail otherwise and as this is a mandatory field I wouldn't change this.